src/input/types.js

```diff
@@ -38,6 +38,8 @@ export type SharedPropsT = {
   $disabled: boolean,
   /** Renders UI in 'error' state */
   $error: boolean,
+  /** Renders UI in 'positive' state */
+  $positive: boolean,
```

Comment:
what happens if I use both `$positive` and `$negative` at the same time?

Reply:
it seems that `$error` wins, a warning could be nice?

Reply:
Good point, I'll add the warning. Might be worth using a `STATE` enum or something if these get more complicated.